pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[build-system]
+requires = ["flit_scm"]
+build-backend = "flit_scm:buildapi"
+
+[project]
+name = "wheel"
+description = "A built-package format for Python"
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: System :: Archiving :: Packaging",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10"
+]
+authors = [{name = "Daniel Holth", email = "dholth@fastmail.fm"}]
+maintainers = [{name = "Alex Grönholm", email = "alex.gronholm@nextday.fi"}]
+keywords = ["wheel", "packaging"]
+license = {file = "LICENSE.txt"}
+requires-python = ">=3.7"
+dependencies = [
+    "setuptools >= 45.2.0"
+]
+dynamic = ["version"]
+
+[project.urls]
+Documentation = "https://wheel.readthedocs.io/"
+Changelog = "https://wheel.readthedocs.io/en/stable/news.html"
+"Issue Tracker" = "https://github.com/pypa/wheel/issues"
+
+[project.scripts]
+wheel = "wheel.cli:main"
+
+[project.entry-points."distutils.commands"]
+bdist_wheel = "wheel.bdist_wheel:bdist_wheel"
+
+[project.optional-dependencies]
+test = [
+    "pytest >= 3.0.0"
+]
+
+[tool.flit.sdist]
+include = ["src/wheel/_version.py"]
+exclude = [
+    ".cirrus.yml",
+    ".github/*",
+    ".gitignore",
+    ".pre-commit-config.yaml",
+    ".readthedocs.yml"
+]
+
+[tool.setuptools_scm]
+write_to = "src/wheel/_version.py"
+
+[tool.black]
+target-version = ['py37']
+extend-exclude = '''
+^/src/wheel/vendored/
+'''
+
+[tool.isort]
+src_paths = ["src"]
+profile = "black"
+skip_gitignore = true
+
+[tool.flake8]
```

Comment:
Nope. You can use .flake8 or put it in tox, but no. The flake8 maintainers are not interested in toml.

Reply:
That is why I use pyproject-flake8, not regular flake8.

Reply:
Ahh, didn't see that, cool!